datafusion/optimizer/src/optimize_projections/mod.rs

```diff
@@ -36,7 +36,7 @@ use datafusion_expr::{
     TableScan, Window,
 };
 
-use crate::optimize_projections::required_indices::RequiredIndicies;
+use crate::optimize_projections::required_indices::RequiredIndices;
```

Comment:
Since this is an internal struct (not exposed publically) I think this is not an API change: https://docs.rs/datafusion/latest/datafusion/index.html?search=RequiredIndicies